src/i-output.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Interface for output state.
+ */
+export type IOutput = any;
+
+/*
+NBFORMAT SCHEMA
```

Comment:
:+1: - we could then use it for doing verification at runtime (in addition) or (instead) of compile time.